QueryBuilder/Typed/FilteredQuery.cs

```diff
@@ -337,7 +349,7 @@ private WhereScalarFunctionCondition CreateWhereScalarCondition<TModel>(string p
             };
         }
 
-        private WhereComparisonCondition CreateWhereComparisonCondition<TModel>(string propertyName, ComparisonOperators op, object value, Type type, string alias)
+        internal WhereComparisonCondition CreateWhereComparisonCondition<TModel>(string propertyName, ComparisonOperators op, object value, Type type, string alias)
```

Comment:
Did this need to change visibility from private to internal? I don't see it being used outside this class.

Reply:
Good eye. Not anymore.